src/components/pages/admin/events/dashboard/fanNotifications/Index.js

```diff
@@ -127,21 +125,66 @@ class Index extends Component {
 			scheduledAt: null,
 			scheduleProgress: null,
 			scheduleSent: null,
-			isSchedule: false,
-			updateNotification: false,
 			broadcastId: null,
 			isNotificationAfter: true,
 			isEventEnded: false,
 			datesOptions: [],
-			hasEventStarted: true
+			hasEventStarted: true,
+			count: 1,
+			inProgress: false
 		};
 	}
 
 	componentDidMount() {
+		const { broadcastSent, scheduleProgress, scheduleSent } = this.state;
 		//TODO check if the event is running before enabling the button
 		this.setState({ canTrigger: true });
 
 		this.loadNotificationDetails();
+
+		!broadcastSent ? this.autoLoadProgress() : null;
+	}
+
+	autoLoadProgress() {
+		this.gradualTimer();
+	}
+
+	gradualTimer = () => {
```

Comment:
Could we not make this a member of the class
```suggestion
	gradualTimer() {
```